packages/fast-components-class-name-contracts-msft/src/progress/index.ts

```diff
@@ -22,30 +22,15 @@ export interface ProgressClassNameContract extends BaseProgressClassNameContract
     /**
      * The indeterminate progress indicator
      */
-    progress_dot?: string;
+    progress_indicator__indeterminate?: string;
 
     /**
      * The indeterminate progress indicator 1 modifier

```

Comment:
Thats not a problem. Its just a terrible name for these new elements. Also only 2 now should instead of 5... I assume we still want to keep all the progress_dot__X even though they no longer have elements?

Reply:
You should keep them and mark them deprecated